mxcube3/ui/actions/general.js

```diff
@@ -152,11 +176,15 @@ export function getInitialStatus() {
     const pchains = [
```

Comment:
I would rather open another issue and keep this one focused on its topic. I can take care myself of course.